ui/lib/core/addon/components/kv-object-editor.hbs

```diff
@@ -11,7 +11,7 @@
     class={{@labelClass}}
     data-test-kv-label={{true}}
   />
-  {{#if @validationError}}
+  {{#if (and @validationError (not @isSingleRow))}}
```

Comment:
But don't you still need to validation error to display if the user attempts to just submit a title?

Reply:
<img width="1140" alt="Screenshot 2024-01-19 at 1 12 56 PM" src="https://github.com/hashicorp/vault/assets/30884335/59223312-9650-421a-af7f-d8f9a7867e62">
This is how it looks if we have both validations shown. The form field has an inline error message as well as the validationError in the kv-object-editor

Reply:
I still don't think this should be hidden based on whether or not it's a singleRow, that conditional isn't relevant. The issue here is both components manage validation. 

Might be worth asking the team, but the validation above the input is anti-pattern...so only one component here should be managing validation. I think the block should be removed all together (since it's also in the wrong place) but that could break a bunch of tests.

Reply:
Sounds good! I'll keep the validations here for now :)

Reply:
We should definitely remove one of them - I think youre just the first to come across this bug!